doc/classes/EditorSyntaxHighlighter.xml

```diff
@@ -13,13 +13,14 @@
 		<method name="_get_name" qualifiers="virtual const">
 			<return type="String" />
 			<description>
-				Virtual method which can be overridden to return the syntax highlighter name.
+				Virtual method which can be overridden to return the syntax highlighter name. This is a name that is displayed in the editor when selecting between syntax highlighters, e.g. [code]Markdown[/code].
 			</description>
 		</method>
 		<method name="_get_supported_languages" qualifiers="virtual const">
 			<return type="PackedStringArray" />
 			<description>
-				Virtual method which can be overridden to return the supported language names.
+				Virtual method which can be overridden to return the supported file extensions. This method should return a list of file extensions without the leading period, e.g. [code]PackedStringArray(["md", "markdown"])[/code].
+				Files that don't have any supported syntax highlighter will not use any syntax highlighting (plain text). The user can manually override the syntax highlighting using [b]Edit &gt; Syntax Highlighter[/b] at the top of the script editor.
```

Comment:
Minor thing. One time a contraction is used (_"don't"_) and the next it is not (_"will not"_).
This sentence is a bit repetitive. I am not sure how important it is to highlight (no pun intended) _"any syntax highlighting"_ here.
I've personally seen _"Script Editor"_ capitalized very often. I believe we capitalize docks & similar, and this is one such case.

```suggestion
				Files that do not have any supported syntax highlighter will be displayed without any highlighting.
				[b]Note:[/b] The user can manually override the syntax highlighting used for any open file, via the [b]Edit &gt; Syntax Highlighter[/b] option at the top of the Script Editor.
```
Or similar